modules/distribution/src/main/bin/synapse.sh

```diff
@@ -146,12 +146,17 @@ if [ "$1" = "-xdebug" ]; then
     SERVER_NAME=$2
     shift 2 # -serverName and actual name
 
+  elif [ "$1" = "-synapseDebug" ]; then
+    SYNAPSE_DEBUG="synapseDebug"
+    shift
+
 elif [ "$1" = "-h" ]; then
     echo "Usage: synapse.sh ( commands ... )"
     echo "commands:"
     echo "  -xdebug            Start Synapse under JPDA debugger"
     echo "  -sample (number)   Start with sample Synapse configuration of given number"
     echo "  -serverName <name> Name of the Synapse server instance"
+    echo "  -synapseName       Start Synapse under debug mode"
```

Comment:
this should be changed to -synapseDebug isn't it?

Reply:
Fixed it. Thanks for pointing out and sorry for the mistake.